### DISCUSSION TITLE
Validator - Checking for heartbeat interval while testing end_session_ack (test 16)

### DISCUSSION THREAD

**Discussion by PavelPu123** [DMTF/libspdm#1829] [Q&A] [answered]
Hello
I'm not sure I understand the logic of checking the heartbeat interval in spdm_test_case_end_session_ack_setup_session (line 164):

[SPDM-Responder-Validator/library/spdm_responder_conformance_test_lib/spdm_responder_test_16_end_session_ack.c](https://github.com/DMTF/SPDM-Responder-Validator/blob/d2451d8533fff6b0249484a4b9bf52705f73e948/library/spdm_responder_conformance_test_lib/spdm_responder_test_16_end_session_ack.c#L164-L166)

Lines 164 to 166 in [d2451d8](https://github.com/DMTF/SPDM-Responder-Validator/commit/d2451d8533fff6b0249484a4b9bf52705f73e948)

 if (test_buffer->heartbeat_period == 0) { 
     return false; 
 } 
This check affects the specific test since it's not a requirement to setup the heartbeat interval in responder as part of this test.

Thanks in advance for your comments and thoughts.

**Comment by jyao1** [comment, marked as answer]
Thanks for the reporting. It is fixed in https://github.com/DMTF/SPDM-Responder-Validator/pull/66
